docs/examples/unsupported-requester.js

```diff
@@ -0,0 +1,69 @@
+import { EventEmitter } from 'events';
+import { URL } from 'url';
+// `npm i --save caseless` although it's available if `request` is installed
+import caseless from 'caseless';
+
+export default function (options) {
+  return new Request(options);
+};
+
+class Request extends EventEmitter {
+  constructor (options) {
+    super();
+    const self = this;
+
+    // All of the properties that I defined are required...
+    self.uri = typeof options.uri === 'string'
+      ? new URL(options.uri) : options.uri;
+
+    // Use options.headers if need to serialize...
+    self.headers = caseless(options.headers);
+
+    // Cloudscraper will only call `request.callback` for the very last request
+    self.callback = options.callback;
+
+    // Perform the actual request and emit events
+    if (undefined !== void 0) {
```

Comment:
What? %)

Reply:
@codemanki It's a pre-es5 check which doesn't make the most since in 2019. It's only there so that branch won't ever be executed. Feel free to adjust things so that they are more clear.

https://stackoverflow.com/a/8783528

Reply:
My IDE complains about *always* true or false conditions:
```
if (false) {
}
```
I was just out smarting my IDE. :P